packages/menu/src/MenuItem/MenuItem.styles.ts

```diff
@@ -77,9 +77,7 @@ export const getMenuItemStyles = ({
 
       // Highlighted
       [css`
-        &,
-        &:hover,
-        &:focus {
+        &:focus-visible {
```

Comment:
This is also breaking some generated stories. We should add some test data-property selector to allow us to test this in SB
i.e. `[data-focus="true"] {...}`